tests/unit/test_install.py

```diff
@@ -0,0 +1,155 @@
+import webbrowser
+from unittest.mock import MagicMock, create_autospec
+
+import pytest
+from databricks.labs.blueprint.installation import Installation, MockInstallation
+from databricks.labs.blueprint.tui import MockPrompts
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors import NotFound
+
+from databricks.labs.remorph.__about__ import __version__
+from databricks.labs.remorph.install import WorkspaceInstallation, WorkspaceInstaller
+
+
+@pytest.fixture
+def ws():
+    ws = create_autospec(WorkspaceClient)
+
+    ws.product = "remorph"
+    ws.product_version = __version__
```

Comment:
These are not exposed as properties of a workspace client. Did you try to run production code? 😉

Reply:
Removed :)